msteams-platform/toolkit/debug-teams-app-visual-studio.md

```diff
@@ -0,0 +1,174 @@
+---
+author: surbhigupta
+title: Debug your Teams app for Visual Studio
+description: In this module, learn how to debug your Teams app locally in Visual Studio using Teams Toolkit
+ms.author: v-amprasad
+ms.localizationpriority: high
+ms.topic: overview
+ms.date: 03/21/2022
+---
+
+# Debug your Teams app locally using Visual Studio
+
+Teams Toolkit helps you to debug and preview your Microsoft Teams app locally. Debug is a process of building, checking, detecting, and correcting issues or bugs in your app. Debug ensures that the program runs successfully. Visual Studio allows you to debug tab, bot, message extension. Teams Toolkit supports the following debug features:
+
+* Prepare Teams app dependencies.
+* Start debugging.
+* Toggle breakpoints.
+* Hot reload.
+* Stop debugging.
+
+During the debug process, Teams Toolkit automatically starts app services, initiates debug, and side loads Teams app. After debug, you can preview the Teams app in Teams web client. You can also customize debug settings to use your bot endpoints, or environment variables to load your configured app.
+
+## Prerequisites
+
+| &nbsp; | Install | For using... |
+| --- | --- | --- |
+| &nbsp; | **Required** | &nbsp; |
+| &nbsp; | Visual Studio | You can install the free community edition of Visual Studio, and install the workload. |
```

Comment:
Visual Studio 2022 v 17.3

Reply:
You can install the enterprise edition of Visual Studio, and install the "ASP.NET "workload and Microsoft Teams Development Tools.

Reply:
These two comments has been implemented.